package.json

```diff
@@ -7,16 +7,16 @@
 		"rbxts-build": "out/index.js"
 	},
 	"scripts": {
-		"build": "tspc",
-		"build-watch": "tspc -w",
+		"build": "tsc",
+		"build-watch": "tsc -w",
 		"prepublishOnly": "npm run build",
 		"devlink": "cd devlink && npm link"
 	},
 	"keywords": [],
-	"author": "osyrisrblx",
+	"author": "CapedBojji",
```

Comment:
You've changed the autor here.

Reply:
My bad you qere taking long so i want gonna upload a linuc version called rbxts-build-linux. Will fix that